docs/blogs/IntroGatewayAPI/GettingCreative.md

```diff
@@ -0,0 +1,294 @@
+## Getting Creative
+
+In the previous section, we showed how to create a simple HTTPRoute. Let’s expand upon this example and include more
+advanced usages of Gateway API.
+
+### Configuring DNS
+
+If you would like the example routes to work in your browser instead of using `curl`, you will need to create a DNS record,
+point it to `INGRESS_HOST`, and configure the HTTPRoute’s `spec.hostnames` and the Gateway’s `spec.listeners[].hostname`
+to support the new domain name:
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1beta1
+kind: Gateway
+...
+spec:
+gatewayClassName: istio
+listeners:
+- name: demo
+  hostname: "*.<DOMAIN_NAME>" [1]
```

Comment:
Depending on what renderer is ultimately used to render the Markdown, you might want to use a comment to ensure the callout marker doesn't get interpreted as part of the yaml.  
```suggestion
  hostname: "*.<DOMAIN_NAME>" # [1]
```

Reply:
Done